doc/classes/Callable.xml

```diff
@@ -112,6 +112,12 @@
 				Calls the method represented by this [Callable]. Unlike [method call], this method expects all arguments to be contained inside the [param arguments] [Array].
 			</description>
 		</method>
+		<method name="get_argument_count" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the total number of arguments of this [Callable], accounting for any bound (or unbound) arguments. The number of bound arguments is subtracted from the total, and the number of unbound arguments are added.
```

Comment:
I understand what this is trying to say but it is a bit confusing when worded like this. I have no suggestions at the moment.

Reply:
Me too, was trying to figure out how to say it

Reply:
Okay, I'll help you think. From the outsider developer's perspective, what changes? What needs to be known? What will this method be used for?

Judging by the code, I would say the important part to get here is that what we mean by "arguments" is "bunch of values that _can_ (oftentimes _should_) be passed into the Callable"
```suggestion
				Returns the total number of arguments this [Callable] should take, including optional arguments.
```
The part about `bind` and `unbind` is a consequence of this definition.

Reply:
I like that, I'd keep the bind/unbind explicit as it can be very, *very* confusing (I got it backwards the first time I wrote the code and added instead of subtracted), but I'll take that suggestion as a base 🙂 thank you!

Reply:
Oh by all means, it needs to be specified to remove any doubts. It's just that -- and I repeat myself -- by rewording the definition this way it makes the behavior of `bind` and `unbind` look like a natural consequence of it, instead of a potentially confusing exception to the norm.